build.py

```diff
@@ -1396,7 +1420,13 @@ def container_build(images, backends, repoagents, endpoints):
         repo_install_dir = os.path.join(FLAGS.build_dir, be, 'install')
 
         mkdir(FLAGS.build_dir)
-        gitclone(FLAGS.build_dir, backend_repo(be), backends[be], be)
+        # If tflite backend, source from external repo for git clone
+        if be == 'tflite':
+            gitclone(FLAGS.build_dir, backend_repo(be), backends[be], be,
+                     'https://gitlab.com/arm-research/smarter/')
```

Comment:
It would be better to set github-organization default to None in the argparse, and then manually set a default value for it based on platform if the user doesn't set it explicitly. Then you don't need any of these org changes.

Reply:
@deadeyegoodwin  Changes are made as per your request, however tying the github org to the platform and not the backend makes it such that the tflite backend can ONLY be built into the `ubuntu/arm64` build, although it should build fine into the jetpack build as well.

Reply:
That's a good point. Ideally we'd have an option to set github org for each backend independently... but not worth doing that at this point. I would say you should just revert fdc20a0 to go back to what you had. That means that someone can't build the armnn backend from their fork without editing build.py but I think that is ok for now.

Reply:
Agreed. `fdc20a0` reverted.